briskedu/brisk-learning
# Add logout menu option

Summary:
Adding options menu with logout as first option

Test Plan:
1. Confirm I can login
2. Confirm trigger shows up on hover
3. Confirm it disappears on click
4. Confirm I can logout
5. Confirm message shows up
![Options Menu](https://user-images.githubusercontent.com/15060005/226711366-82fa5ba8-b4e6-4751-9e5e-54366c7d8e59.gif)
